### ISSUE TITLE
[Crypto] Add Bitcoin Lightning

### ISSUE THREAD

**Issue by trymeouteh** [haveno-dex/listing#24]
- [X] I confirm there are no other open issues about this asset
- [X] I understand that to actually add the asset to Haveno, a PR from the developers/community of the proposed asset might be necessary
- [X] I understand that adding the asset is at discretion of the Haveno Core Team and applying doesn't necessarily imply acceptance

### What's the ticker symbol of the project?
BTC-LN

### Why should it be added to Haveno? How would it benefit the project and the community?
Bitcoin lighting is getting popular, it is fast and has low fees. If Haveno allows banking transactions, no reason to be against bitcoin lightning for it being off chain.

### Is it an Ethereum token (ERC-20)?

No

### Was the coin pre-mined before public launch?

No

### Is there a dev fee for the project maintainers?

No

### Please provide resources about the project (Website, Twitter, etc)
https://lightning.network/

### Additional notes
More BTMs, businesses and exchanges are offering bitcoin lightning 

Lightning maybe more private than on-chain bitcoin, making it great for swapping bitcoin to Monero privately.

**Comment by HavenoDEX** [comment]
Please acknowledge the statements at the top of the template.

**Comment by NotIshanSingh** [comment]
Lightning Network is shit

**Comment by ghost** [comment]
I agree with @SoiledCold5, however bitcoin is still the largest cryptocurrency by userbase and market cap. Lightning encourages relay centralization and corporate capture, but is exchanging bitcoin without lightning even feasible  anymore?

**Comment by peepo5** [comment]
Maybe rather than lightning network, alternate chains?
